static/app/views/organizationGroupDetails/groupActivityItem.tsx

```diff
@@ -140,6 +141,35 @@ function GroupActivityItem({activity, orgSlug, projectId, author}: Props) {
               author,
             });
       case GroupActivityType.SET_RESOLVED_IN_COMMIT:
+        const deployed_releases: Array<BaseRelease> = [];
+        for (const release of activity.data.commit.releases) {
+          if (release.dateReleased !== null) {
+            deployed_releases.push(release);
+          }
+        }
+        if (deployed_releases.length !== 0) {
+          return tct(
+            '[author] marked this issue as resolved in [version]\n' +
+              'This commit was released in [release]',
+            {
```

Comment:
I feel like there might be a better way to do the conditional templating here. Maybe scott has an idea?